src/localization/resources.ts

```diff
@@ -7,7 +11,11 @@ import notifications from '@/public/locales/en/notifications.json';
  */
 export const resources = {
   en: {
+    actionpanel: actionPanel,
     components: components,
+    exoplanetspanel: exoplanetsPanel,
+    flightcontrolpanel: flightcontrolPanel,
+    geolocationpanel: geolocationPanel,
     notifications: notifications
```

Comment:
No comment, just a question about the coding style.   How come it is `actionPanel` here , but `ActionIcon` in the ActionsBreadcrumbs.tsx?

Reply:
My thinking is that each of these JSON files will be for an entire panel, and then the file itself will have some internal structure that highlights the different sub-components. But since the actionPanel does not have much text, I figured nesting things under sub-names would be unnecessary in this case